src/player.ts

```diff
@@ -405,4 +405,18 @@ export class Player extends NativeInstance<PlayerConfig> {
   castStop = () => {
     PlayerModule.castStop(this.nativeId);
   };
+
+  /**
+   * @returns The currently selected video quality.
+   */
+  getVideoQuality = async (): Promise<number> => {
+    return PlayerModule.getVideoQuality(this.nativeId);
+  };
+
+  /**
```

Comment:
Same here

Reply:
Address in cafb79f9ec5399959d523103108a7cf581495c07